app/views/layouts/component.njk

```diff
@@ -33,6 +33,6 @@
   </div>
 
   {% block examples %}
-    {{ showExamples(componentName, componentNameHuman, componentData) }}
+    {{ showExamples(componentName, componentNameHuman, componentData,legacy_query) }}
```

Comment:
It'd be good if the breadcrumbs in this view also preserved the legacy state (by using `legacy_query`)

Reply:
Well spotted, I've added the mode to this breadcrumb and the all-components breadcrumb too.